lib/rubocop/cop/cop.rb

```diff
@@ -212,10 +217,16 @@ def reference_url
         (url.nil? || url.empty?) ? nil : url
       end
 
+      def details
+        details = cop_config && cop_config['Details']
+        (details.nil? || details.empty?) ? 'No extra details set.' : details
```

Comment:
I think that printing `No extra details set.` every time a cop without extra details reports might be a bit too much. I'd prefer adding nothing in those cases.